ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-nightlies-postsubmits.yaml

```diff
@@ -228,6 +228,52 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^nightlies$
+    cluster: build04
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+      timeout: 5h0m0s
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
```

Comment:
@gavin-stackrox should `max_concurrency` be bumped here too, as well as in `branch-ci-stackrox-stackrox-nightlies-gke-ui-e2e-tests`?